tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -147,69 +155,136 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 
 	var _ = Describe("Post-release verification", func() {
 
-		It("verifies that a PipelineRun is created in dev namespace.", func() {
+		It("verifies that a PipelineRuns are created in dev namespace.", func() {
 			Eventually(func() bool {
-				prList, err := fw.AsKubeAdmin.TektonController.ListAllPipelineRuns(devNamespace)
-				if err != nil || prList == nil || len(prList.Items) < 1 {
+				buildPr, err := fw.AsKubeAdmin.HasController.GetComponentPipelineRun(componentName, applicationName, devNamespace, "")
+				if err != nil {
+					GinkgoWriter.Println("PipelineRun has not been created yet")
+					return false
+				}
+				buildPrName = buildPr.Name
+
+				additionalBuildPr, err := fw.AsKubeAdmin.HasController.GetComponentPipelineRun(additionalComponentName, applicationName, devNamespace, "")
+				if err != nil {
+					GinkgoWriter.Println("PipelineRun has not been created yet")
 					return false
 				}
 
-				return strings.Contains(prList.Items[0].Name, componentName)
+				buildPrName = buildPr.Name
+				additionalBuildPrName = additionalBuildPr.Name
+
+				GinkgoWriter.Printf("\nFirst Build PipelineRun:\n %s", buildPrName)
+				GinkgoWriter.Printf("\nSecond Build PipelineRun:\n %s", additionalBuildPrName)
+
+				return strings.Contains(buildPrName, "release") && strings.Contains(additionalBuildPrName, "release")
```

Comment:
[`fw.AsKubeAdmin.HasController.GetComponentPipelineRun()`](https://github.com/redhat-appstudio/e2e-tests/blob/7fdcffcec67a5171305f744f9e288a24d485c5f1/pkg/utils/has/controller.go#L424) returns a pipelinerun that is bound to the component name specified in the first parameter. if such pipelinerun does not exists, it returns an error.

Thus these lines (173-179) are not necessary - instead `return true` here. 
that's the main reason why I was suggesting to use that method, because it makes the code shorter.

Reply:
agree for the return part, yet the assignment lines are to be able to use the  global variables  `additionalBuildPrName` and `buildPrName`, it's used in test after

Reply:
or use the `fw.AsKubeAdmin.HasController.GetComponentPipelineRun` again, then it's not needed 😜

Reply:
done